src/master-playlist-controller.js

```diff
@@ -956,6 +956,8 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.trigger('error');
       return;
     }
+
+    this.pauseLoading();
```

Comment:
Are we sure we want to abort and pause all loaders if we're blacklisting one?

Reply:
Good question. I think that we may be able to narrow when/what we pauseLoading during a blacklist. For instance this function can be used to blacklist things other than the current playlist and we should only pauseLoading if we are blacklisting the current playlist. It's also possible that we should only stop the audio loaders when the audio group is actually changing. Let me see what I can do.

Reply:
Updated the code to do the following:
1. Check if the audio/subtitle group changed, if it did then we know that we need to pause loading.
2. Verify that `currentPlaylist` is actually the `media`, as its possible that it isn't